### DISCUSSION TITLE
Great opportunity to add more functionality

### DISCUSSION THREAD

**Discussion by spences10** [spences10/svead#134] [Ideas]
The [Astro SEO component] looks like a really good resource to help with some additional missing features for this component.

There's a lot of really good information detailed on the [README]

[Astro SEO component]: https://github.com/jonasmerlin/astro-seo/
[readme]:https://github.com/jonasmerlin/astro-seo/blob/main/README.md

**Comment by yeshealer** [comment]
This opportunity is totally free? Or paid work?
@spences10

**Reply by spences10** [reply]
It's for contributions @yeshealer, I've implemented a lot of the features/pointers from there myself and it's there as a discussion point and as a reminder to me.

I'm not paying people to work on it. Thanks.
